src/autogluon/bench/datasets/multimodal_dataset.py

```diff
@@ -108,8 +129,15 @@ def __init__(self, split="train"):
         self._path = os.path.join(get_data_home_dir(), "shopee")
         load_zip.unzip(self._INFO["data"]["url"], unzip_dir=self._path, sha1sum=self._INFO["data"]["sha1sum"])
         self._base_folder = os.path.join(self._path, "shopee")
-        self._data = pd.read_csv(os.path.join(self._base_folder, f"{self._split}.csv"))
-        self._data["image"] = self._data["image"].apply(lambda ele: _path_expander(ele, base_folder=self._base_folder))
+        try:
+            data_path = os.path.join(self._base_folder, f"{self._split}.csv")
+            self._data = pd.read_csv(data_path)
+            self._data["image"] = self._data["image"].apply(
+                lambda ele: _path_expander(ele, base_folder=self._base_folder)
+            )
+        except FileNotFoundError as e:
+            logger.warn(e)
```

Comment:
So we will continue the benchmark even if the data file is not found?

Reply:
Will skip the rest

Reply:
Yes. So the idea is for the dataloader to be able to load any data split. If the data can't be downloaded or opened, we assign the data to be None. We will rely on the predictor to handle the empty data and throw error.